.github/workflows/publish-npm.yml

```diff
@@ -3,6 +3,9 @@ name: Publish to npm
 on:
   workflow_dispatch:
   workflow_call:
+    secrets:
+      NPMJS_PUBLISH_ITWIN:
+        required: true
```

Comment:
Will this affect workflow_dispatch in any way? Don't want manual publish to break

Reply:
It shouldnt, I added it only for `workflow_call`